crates/ruma-client-api/src/uiaa/get_uiaa_fallback_page.rs

```diff
@@ -39,6 +40,10 @@ pub mod v3 {
     #[response(error = crate::Error)]
     #[derive(Default)]
     pub struct Response {
+        /// Content type of the body.
```

Comment:
Great, the only reason for this proposed change is as I'm trying to make these endpoints play nicely with Conduit, to avoid the following pattern which is found in routes that don't use the `ruma_route` macro:

```rust
let ok = match ok {
  Ok(x) => x,
  Err(e) => return e.into_response(),
}
```

Reply:
I am not sure if you misunderstood since this is closed as "resolved", but what we mean is that we would like to fix this in Ruma, but the solution is to not use the `response` attribute macro and instead implement the response traits manually. `Response` should probably be an enum for whether the server wants to redirect or serve a page.

Reply:
I wrote #1797 to fix this, so I am closing this PR.